src/controllers/update/update.beneficial.owner.gov.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { NextFunction, Request, Response } from "express";
+import { logger } from "../../utils/logger";
+import { UPDATE_BENEFICIAL_OWNER_GOV_PAGE, UPDATE_BENEFICIAL_OWNER_TYPE_URL } from "../../config";
+import { getBeneficialOwnerGov, getBeneficialOwnerGovById, postBeneficialOwnerGov, removeBeneficialOwnerGov, updateBeneficialOwnerGov } from "../../utils/beneficial.owner.gov";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  logger.debugRequest(req, `${req.method} ${req.route.path}`);
```

Comment:
Is this logger line required? Doesnt it already exist in `getBeneficialOwnerGov`?